api/kubernetes/helm-chart/kube-prometheus-stack.yaml

```diff
@@ -0,0 +1,72 @@
+namespace: prometheus
```

Comment:
This file should not be in helm-chart directory, it does not belong to the chart. Upper directory maybe.

Reply:
Moved to upper directory